src/lib/featureFlagSlice.ts

```diff
@@ -0,0 +1,41 @@
+import { createSlice, PayloadAction } from "@reduxjs/toolkit";
+
+// const API_ADDRESS = "http://localhost:8080";
+
+interface FeatureFlagsState {
+  enableAbout: boolean;
+  enableWiki: boolean;
+  enableFAQ: boolean;
+
+  // Add more feature flags as needed
+}
+
+export const initialState: FeatureFlagsState = {
+  enableAbout: false,
+  enableWiki: false,
+  enableFAQ: false,
+
+  // Initialize other feature flags
+};
+
+const featureFlagSlice = createSlice({
+  name: "featureFlags",
+  initialState,
+  reducers: {
+    toggleFeatureFlag: (
+      state,
+      action: PayloadAction<keyof FeatureFlagsState>
+    ) => {
+      const flagKey = action.payload;
+      if (flagKey in state) {
+        state[flagKey] = !state[flagKey];
+      }
+    },
```

Comment:
I'm a little worried about toggling, because it is not idempotent (and also because redux is async). 
Maybe the action should take in both `flagKey` and `newState`?

Reply:
I don't understand the idea here, the values can only be "on" or "off", why does toggling not make sense?